llvm/test/CodeGen/RISCV/fold-binop-into-select.ll

```diff
@@ -58,3 +58,15 @@ entry:
   %res = sub i64 2, %select_
   ret i64 %res
 }
+
+define i64 @fold_binop_into_select_return_constant(i1 %c) {
```

Comment:
Please make sure that this function cannot be folded by middle-end.
See: https://godbolt.org/z/h1M7frG4j

Reply:
Sorry, it is fine for crash fixes.